sdk/inc/azure/core/az_span.h

```diff
@@ -57,8 +59,9 @@ AZ_NODISCARD AZ_INLINE int32_t az_span_size(az_span span) { return span._interna
 /**
  * @brief Returns an #az_span over a byte buffer.
  *
- * @param[in] ptr The memory address of the 1st byte in the byte buffer.
- * @param[in] size The number of total bytes in the byte buffer.
+ * @param[in] ptr Memory address of the first byte in the byte buffer.
+ * @param[in] size Total number of bytes in the byte buffer.
```

Comment:
Why did we reword this? I prefer the original.

Reply:
Maybe it's my English, but I've seen "total number" everywhere but haven't seen "number of total". With previous wording, it sounds like "total" is attribute of a byte ("total byte"), and not an attribute of "number" (i.e. not "total number", but `number_of(total_bytes)`). I.e. previous is `count(sum(bytes))`, new is `sum(count(bytes))`.

Book ngrams: https://books.google.com/ngrams/graph?content=total+number+of%2Cnumber+of+total&year_start=1800&year_end=2019&corpus=26&smoothing=0

Search trends: https://trends.google.com/trends/explore?date=all&q=%22total%20number%20of%22,%22number%20of%20total%22

Reply:
Sorry, let me clarify. Yo are right that it should be `total number of bytes` and **not** `number of total bytes`

I meant the following, for both ptr and size (keep the `the`):
```C
* @param[in] ptr The memory address of the first byte in the byte buffer.
 * @param[in] size The total number of bytes in the byte buffer.
```